core/src/main/resources/hudson/model/AbstractBuild/tasks.jelly

```diff
@@ -30,7 +30,6 @@ THE SOFTWARE.
   <j:choose>
     <j:when test="${newBuildPage}">
       <l:task href="${buildUrl.baseUrl}/changes" icon="symbol-changes" title="${%Changes}"/>
-      <p:console-link/>
```

Comment:
is this intended to be removed?

Reply:
Yeah, it's in the tab bar now + this bit of code is inside the `newBuildPage` condition.